js/binancetr.js

```diff
@@ -0,0 +1,277 @@
+        //                        "filters":[
+        //                            {"filterType":"PRICE_FILTER","minPrice":"0.01000000","maxPrice":"10000.00000000","tickSize":"0.01000000","applyToMarket":false},
+        //                            {"filterType":"PERCENT_PRICE","multiplierUp":5,"multiplierDown":0.2,"avgPriceMins":"5","applyToMarket":false},
+        //                            {"filterType":"LOT_SIZE","minQty":"0.10000000","maxQty":"9222449.00000000","stepSize":"0.10000000","applyToMarket":false},
+        //                            {"filterType":"MIN_NOTIONAL","avgPriceMins":"5","minNotional":"10.00000000","applyToMarket":true},
+        //                            {"filterType":"ICEBERG_PARTS","applyToMarket":false,"limit":"10"},
+        //                            {"filterType":"MARKET_LOT_SIZE","minQty":"0.00000000","maxQty":"191987.27930555","stepSize":"0.00000000","applyToMarket":false},
+        //                            {"filterType":"MAX_NUM_ORDERS","applyToMarket":false},
+        //                            {"filterType":"MAX_NUM_ALGO_ORDERS","applyToMarket":false,"maxNumAlgoOrders":"5"}
+        //                        ],
+        //                        "orderTypes":[
+        //                            "LIMIT",
+        //                            "LIMIT_MAKER",
+        //                            "MARKET",
+        //                            "STOP_LOSS_LIMIT",
+        //                            "TAKE_PROFIT_LIMIT"
+        //                            ],
+        //                        "icebergEnable":1,
+        //                        "ocoEnable":1,
+        //                        "spotTradingEnable":1,
+        //                        "marginTradingEnable":0,
+        //                        "permissions":["SPOT"]
+        //                    },
+        //                ]
+        //            }
+        //        }
+        //
+        const result = [];
+        for (let i = 0; i < response['data']['list'].length; i++) {
+            const market = response['data']['list'][i];
+            // const type = this.safeInteger(market, 'type'); // Important to see endpoint for this market
+            const id = this.safeString (market, 'symbol');
+            const filterType0 = market['filters'][0];
+            const filterType2 = market['filters'][2];
+            const minPrice = this.safeValue (filterType0, 'minPrice');
+            const maxPrice = this.safeValue (filterType0, 'maxPrice');
+            const minQty2 = this.safeValue (filterType2, 'minQty');
+            const maxQty2 = this.safeValue (filterType2, 'maxQty');
+            const baseId = this.safeString (market, 'baseAsset');
+            const quoteId = this.safeString (market, 'quoteAsset');
+            const base = this.safeCurrencyCode (baseId);
+            const quote = this.safeCurrencyCode (quoteId);
+            const spotPer = market['permissions'].includes ('SPOT');
```

Comment:
we cannot use `.includes` since it will fail to be properly transpiled, instead we can check using ```(string.indexOf (needle) > -1)```

Reply:
It build ok and didnt give me any transpiled error. But ok, I changed it just in case. Done.